### PR TITLE
feat(cli): display compliance status of cloud accounts/projects/subscriptions

### PR DESCRIPTION
**Pull request by afiune**
## Summary

When working with compliance reports, it is useful to include the status in the output of the **compliance list commands** for each cloud to enable the user to better understand what they are able to target from the Lacework CLI.

This PR is adding a new column `STATUS` to the following commands:

* `laceowrk compliance aws list`
* `lacework compliance gcp list`
* `lacework compliance azure list`

## How did you test this change?

Added unit tests and updated integration tests.

Ran `lacework compliance aws list` on test account:
```
  AWS ACCOUNT    STATUS
---------------+----------
  123456789012   Enabled
  123456789013   Enabled
```
Ran `lacework compliance gcp list` on test account:
```
  ORGANIZATION ID           PROJECT ID            STATUS
------------------+-----------------------------+----------
  n/a               gcr-jenkins-sandbox-274317    Enabled
```
Ran `lacework compliance az list` on test account:
```
              AZURE TENANT                        AZURE SUBSCRIPTION            STATUS
---------------------------------------+--------------------------------------+----------
  123abc-1234-abcd-11222-abcd1234xyzz1   123abc-1234-abcd-11222-abcd1234xyzz1   Enabled
```
## Issue
https://lacework.atlassian.net/browse/ALLY-712

### CODE REVIEW

File: cli/cmd/compliance_aws.go [hunk 1]
```diff
@@ -39,46 +39,14 @@ var (
 		Long:    `List all AWS accounts configured in your account.`,
 		Args:    cobra.NoArgs,
 		RunE: func(_ *cobra.Command, _ []string) error {
+			cli.StartProgress(" Fetching compliance information...")
```
Comment (dmurray-lacework) [inline cli/cmd/compliance_aws.go:42]: nit: spinner text is a little generic. Could make it say 'fetching Aws Accounts'

**Review by dmurray-lacework — APPROVED** [review]
Approved, 1 nitpick but feel free to ignore

File: cli/cmd/compliance_aws.go (hunk 1 above)
Comment (rmoles) [inline cli/cmd/compliance_aws.go:42]:
```suggestion
			cli.StartProgress(" Fetching list of configure AWS accounts...")
```

Comment (afiune) [reply cli/cmd/compliance_aws.go:42]: Since you two asked to change this progress, will change it for the three clouds.  💯

Comment (afiune) [reply cli/cmd/compliance_aws.go:42]: shouldn't the message say "Fetching list of **configured** AWS accounts"

File: cli/cmd/compliance_azure_test.go [hunk 2]
```diff
@@ -84,3 +84,84 @@ func TestSplitAzureSubscriptionsApiResponse(t *testing.T) {
...
+		})
+		expectedJSON := `{
+  "azure_subscriptions": []
+}
+`
+		assert.Equal(t, expectedJSON, cliJSONOutput)
+	})
+}
+
+/*
```
Comment (rmoles) [inline cli/cmd/compliance_azure_test.go:108]: Should these tests be commented out?

Comment (afiune) [reply cli/cmd/compliance_azure_test.go:108]: Yes. I haven't figured out how to mock API calls in our unit test framework. I don't want to remove these tests that work once we figure that out. would that be ok? or should I remove them?

Comment (rmoles) [reply cli/cmd/compliance_azure_test.go:108]: Got it. Could we perhaps leave a TODO or some sort of note on the reasoning?

**Review by rmoles — APPROVED** [review]
Nice! LGTM

File: cli/cmd/compliance_azure_test.go (hunk 2 above)
Comment (afiune) [reply cli/cmd/compliance_azure_test.go]: I found a way to uncomment and run these tests, no need to add a TODO. I used the fake API sever go internal package. 💯

**Pull request merged**